src/stores/grid_renderer_store.ts

```diff
@@ -57,20 +56,14 @@ import { RendererStore } from "./renderer_store";
 export const CELL_BACKGROUND_GRIDLINE_STROKE_STYLE = "#111";
 
 export class GridRenderer {
+  mutators = [] as const;
```

Comment:
is it mandatory to declare for stores? the typing seems to miss this.

Reply:
Does a store makes sense without mutators ?

Reply:
> Does a store makes sense without mutators ?

currently yes because other stores need to register themselves to this store